packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -966,7 +936,8 @@
 					"$(OTHER_CFLAGS)",
 					"-DFOLLY_NO_CONFIG",
 					"-DFOLLY_MOBILE=1",
-					"-DFOLLY_USE_LIBCPP=1 -DFOLLY_CFG_NO_COROUTINES=1",
+					"-DFOLLY_USE_LIBCPP=1",
+					"-DFOLLY_CFG_NO_COROUTINES=1",
```

Comment:
I left these in because it happens every time I run `pod install` and seems to be mostly formatting.